specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2019-08-01/notebook.json

```diff
@@ -0,0 +1,384 @@
+				],
+				"description": "Creates the notebook workspace for a Cosmos DB account.",
+				"operationId": "NotebookWorkspaces_Create",
+				"x-ms-long-running-operation": true,
+				"parameters": [{
+						"$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/subscriptionIdParameter"
+					},
+					{
+						"$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/resourceGroupNameParameter"
+					},
+					{
+						"$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/apiVersionParameter"
+					},
+					{
+						"$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/accountNameParameter"
+					},
+					{
+						"$ref": "#/parameters/NotebookWorkspaceNameParameter"
+					},
+					{
+						"name": "createParameters",
+						"in": "body",
+						"required": true,
+						"schema": {
+							"$ref": "#/definitions/NotebookWorkspaceCreateParameters"
+						},
+						"description": "The notebook workspace to create for the current database account."
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "Creation of notebook workspace will complete asynchronously.",
+						"schema": {
+							"$ref": "#/definitions/NotebookWorkspace"
+						}
+					}
+				},
+				"x-ms-examples": {
+					"CosmosDBNotebookWorkspaceCreate": {
+						"$ref": "./examples/CosmosDBNotebookWorkspaceCreate.json"
+					}
+				}
+			},
+			"delete": {
+				"tags": [
+					"NotebookWorkspacesResource"
+				],
+				"description": "Deletes the notebook workspace for a Cosmos DB account.",
+				"operationId": "NotebookWorkspaces_Delete",
+				"x-ms-long-running-operation": true,
+				"parameters": [{
+						"$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/subscriptionIdParameter"
+					},
+					{
+						"$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/resourceGroupNameParameter"
+					},
+					{
+						"$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/apiVersionParameter"
+					},
+					{
+						"$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/accountNameParameter"
+					},
+					{
+						"$ref": "#/parameters/NotebookWorkspaceNameParameter"
+					}
+				],
+				"responses": {
+					"202": {
+						"description": "The notebook workspace delete operation will complete asynchronously."
+					},
+					"204": {
+						"description": "The specified notebook workspace does not exist in the database account."
+					}
+				},
+				"x-ms-examples": {
+					"CosmosDBNotebookWorkspaceDelete": {
+						"$ref": "./examples/CosmosDBNotebookWorkspaceDelete.json"
+					}
+				}
+			}
+		},
+		"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/notebookWorkspaces/{notebookWorkspaceName}/getConnectionInfo": {
+			"post": {
+				"tags": [
+					"NotebookWorkspacesResource"
+				],
+				"operationId": "NotebookWorkspaces_GetConnectionInfo",
+				"x-ms-examples": {
+					"CosmosDBNotebookWorkspaceGetConnectionInfo": {
```

Comment:
>					 [](start = 0, length = 5)

change tabs-> 4 spaces

Reply:
I think it fixed this last time I ran prettier.